sklearn/metrics/_ranking.py

```diff
@@ -1533,7 +1533,8 @@ def ndcg_score(y_true, y_score, *, k=None, sample_weight=None, ignore_ties=False
     ----------
     y_true : ndarray of shape (n_samples, n_labels)
         True targets of multilabel classification, or true scores of entities
-        to be ranked.
+        to be ranked. Non-negative values in y_true may result in an output
```

Comment:
do you mean "Negative values in `y_true`" rather than "Non-negative"? and maybe the docstring could be a bit more explicit that negative values are deprecated and will result in an error in the future?

Reply:
Right, I'll change the docstring to reflect this more clearly then, thanks for the feedback